lib/ansible/modules/cloud/docker/docker_swarm_service.py

```diff
@@ -2173,6 +2270,17 @@ def _detect_healthcheck_start_period(client):
     return False
 
 
+def _detect_mount_tmpfs_usage(client):
+    for mount in client.module.params['mounts'] or []:
+        if mount.get('type') == 'tmpfs':
+            return True
+        if mount.get('tmpfs_size'):
```

Comment:
Maybe better test
```suggestion
        if mount.get('tmpfs_size') is not None:
```
? (same for next)